### ISSUE TITLE
OneHotEncoding not supported by nn.CrossEntropyLoss()

### ISSUE THREAD

**Issue by JulianneCai** [JulianneCai/fox-brain-cnn-bot#1]
So it turns out that `nn.CrossEntropyLoss()` expects an integer target tensor as the label, so one-hot encoding does not work. One way around is to force it to take an one-hot encoded label by writing a wrapper function for it, but the easier way to do it is to probably just use `LabelEncoder` from `sklearn`.

**Comment by JulianneCai** [comment]
Fixed it in latest commit. Changes were made to `db.py` and `processImage.py`
